difflib/difflib.go

```diff
@@ -877,9 +877,8 @@ func WriteUnifiedDiff(writer io.Writer, diff LineDiffParams) error {
 		equalFormat := " %s"
 
 		if diff.Colored {
-			addFormat = "\x1b[32m+%s"    // Foreground Green
-			removeFormat = "\x1b[31m-%s" // Foreground Red
-			equalFormat = "\x1b[0m %s"   // No Color
+			addFormat = "\x1b[32m+%s\x1b[0m"    // Foreground Green
```

Comment:
what does this do, reset the color?

Reply:
Yes it removes the color code. Since this is a line level diff we should be able to add and remove the color code for each write and have a line level color code. We need to remove it to cover the case where the diff hits EOF and no additional context lines would be written. In that case, we don't want to leave the foreground color in the + or - color or the next addition to that text buffer/string will also be in that color.

Reply:
got it, agreed